CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## [Unreleased]
 ### Added
+- Command Generate - Add  "Application host" parameter (`-p, --application-host`) to `lumber generate`.
```

Comment:
[Conventions] It is a very small detail, but we tend to accumulate changes from top to bottom. So I would move this change at the end of the `### Added` section.

Reply:
`-p, --application-host` ? 🤔
With a strong "auto review" you would have detected this, before submitting this PR to your reviewer.